src/Directory.Build.props

```diff
@@ -39,6 +39,8 @@
 	
 	<UNO_UWP_BUILD>true</UNO_UWP_BUILD>
 	<DefineConstants Condition="'$(UNO_UWP_BUILD)'!='true'">$(DefineConstants);HAS_UNO_WINUI</DefineConstants>
+	<DefineConstants Condition="$(UnoTargetFrameworkOverride.ToLowerInvariant().StartsWith('monoandroid')) OR $(UnoTargetFrameworkOverride.ToLowerInvariant().EndsWith('-android'))">$(DefineConstants);CURRENT_PLATFORM_ANDROID</DefineConstants>
```

Comment:
We cannot use `-android` as `EndsWith` for net6 (this is currently incorrect and will change soon). Let's use StartsWith(`net6.0-android`) for now.